test/e2e/data/capi-operator/capi-providers.yaml

```diff
@@ -12,6 +12,7 @@ metadata:
 spec:
   name: docker
   type: infrastructure
+  version: v1.7.0
```

Comment:
This needs to be checked that it's working

Reply:
Checked this, seems to be working. Which is good, as by default we still install 1.4.6 but users can upgrade to latest using default repo index in CAPI. Will not work for providers which are not added to CAPI, like elemental or CAAPF, k3s, etc. However we need to keep `clusterctl.yaml` config map on 1.4.6 because this updated version will break rancher provisioning.
```
│ NAMESPACE↑                      NAME                     TYPE               PROVIDERNAME      INSTALLEDVERSION      PHASE         │
│ capi-system                     cluster-api              core               cluster-api       v1.7.2                Ready
```